handlers/jaeger.go

```diff
@@ -47,20 +96,14 @@ func getJaegerInfo(requestToken string) (*models.JaegerInfo, int, error) {
 		return nil, http.StatusServiceUnavailable, errors.New("Jaeger URL in cluster is not set in Kiali configuration")
 	}
 
-	internalURL, err := business.GetJaegerInternalURL("/api/services")
-	if err != nil {
-		return nil, http.StatusServiceUnavailable, errors.New("wrong format for Jaeger URL: " + err.Error())
-	}
-	apiURL := internalURL.String()
-
 	// Be sure to copy config.Auth and not modify the existing
 	auth := jaegerConfig.Auth
 	if auth.UseKialiToken {
 		auth.Token = requestToken
 	}
 
-	if ha, err := canAccessURL(apiURL, &auth); !ha {
-		return nil, http.StatusServiceUnavailable, err
+	if ha, _ := canAccessURL(jaegerConfig.InClusterURL, &auth); !ha {
```

Comment:
why not keeping the error detail? I guess it could give some valuable details to troubleshoot?